scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamOp.scala

```diff
@@ -52,6 +57,10 @@ sealed abstract class BeamOp[+A] {
     parDo(FlatMapFn(f))
 }
 
+private final case class SerializableComparator[T](comp: Comparator[T]) extends Comparator[T] {
```

Comment:
note, you may want to use 
https://github.com/twitter/scalding/blob/a9500fd154fa3a318e2fad063e6d3d15e70e8b17/scalding-core/src/main/scala/com/twitter/scalding/serialization/Externalizer.scala#L29

(and in other locations), e.g.
```scala
private class SerializableComparator[A](comp: Comparator[A]) extends Comparator[A] {
  private[this] val extCmp = Externalizer(comp)
  override def compare(o1: T, o2: T): Int = extCmp.get.compare(o1, o2)
}
```
The `Externalizer` tries both java serialization and kryo serialization. You may want to use it for some of the beamops as well with the functions (scalding on cascading does use Externalizer).